4_predictModelToStudyArea.R

```diff
@@ -37,6 +37,7 @@ setwd(loc_envVars)
 db <- dbConnect(SQLite(),dbname=nm_db_file)
 SQLQuery <- "select gridName, fileName from lkpEnvVars;"
 evs <- dbGetQuery(db, SQLQuery)
+evs$gridName <- tolower(evs$gridName)
```

Comment:
> Hmm, gridname might not always be lowercase. But I can make it so if you think that's important.

This was why I was wondering; I had to make this change because `stackOrder` names were all lowercase, but lkpEnvVars$gridName are not, so I got an error because there were unmatched variables before I added this. I don't think it's necessary to make them all lowercase; this should cover it.